crates/nu-command/src/commands/config/command.rs

```diff
@@ -22,20 +22,21 @@ impl WholeStreamCommand for Command {
     }
 
     async fn run(&self, args: CommandArgs) -> Result<OutputStream, ShellError> {
-        let name = args.call_info.name_tag.clone();
+        let name = args.call_info.name_tag;
```

Comment:
This command explanatory shows how `ctx.configs` can be used and accessed. I think it makes the code clearer :)

Other commands are not adapted.